api/Apis/StagingTablesApi.md

```diff
@@ -0,0 +1,36 @@
+# StagingTablesApi
+
+All URIs are relative to *http://localhost:8080/api/2.1/unity-catalog*
+
+| Method | HTTP request | Description |
+|------------- | ------------- | -------------|
+| [**createStagingTable**](StagingTablesApi.md#createStagingTable) | **POST** /staging-tables | Create a staging table |
```

Comment:
Should we put this in `TablesApi` instead?

Reply:
Not sure about this either, i put it under a different tag because it has a different base path but it could still make sense to have it under the Tables tag, @rameshchandra what is your opinion on this?

Reply:
Putting it under tables sounds fine, as this API is only used for managed table creation.